internal/pkg/cli/interfaces.go

```diff
@@ -72,6 +72,7 @@ type workloadListWriter interface {
 
 type applicationStore interface {
 	applicationCreator
+	applicationUpdator
```

Comment:
nit:
```suggestion
	applicationUpdater
```
though apparently Unix likes it spelled with an 'o': https://groups.google.com/g/alt.language.english.spelling.reform/c/qei7oJO1wl0?pli=1